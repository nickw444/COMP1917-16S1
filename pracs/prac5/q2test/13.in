13
3 5 -6 3 9 9 5 1 -1 -8 -7 2 -9
