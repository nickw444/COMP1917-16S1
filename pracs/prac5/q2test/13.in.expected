average of array: 0.461538
